pages/table/expandable-rows-performance.page.tsx

```diff
@@ -0,0 +1,95 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useContext, useState } from 'react';
+import Button from '~components/button';
+import Header from '~components/header';
+import Table from '~components/table';
+import ScreenshotArea from '../utils/screenshot-area';
+import { allInstances } from './expandable-rows/expandable-rows-data';
+import { createColumns } from './expandable-rows/expandable-rows-configs';
+import { useCollection } from '@cloudscape-design/collection-hooks';
+import AppContext, { AppContextType } from '../app/app-context';
+import { Checkbox, SpaceBetween } from '~components';
+
+const allExpandableInstances = allInstances.filter(i => i.children > 0);
+
+type PageContext = React.Context<
+  AppContextType<{
+    expandableRows: boolean;
+  }>
+>;
+
+const columnDefinitions = createColumns();
+
+export default function App() {
+  const { urlParams, setUrlParams } = useContext(AppContext as PageContext);
+  const [isActive, setIsActive] = useState(false);
+  const { items, collectionProps, actions } = useCollection(allInstances, {
+    sorting: {},
+    selection: { trackBy: 'name' },
+    expandableRows: urlParams.expandableRows
+      ? {
+          getId: item => item.name,
+          getParentId: item => item.parentName,
+          defaultExpandedItems: allExpandableInstances,
+        }
+      : undefined,
+  });
+  return (
+    <ScreenshotArea>
+      <h1>Table with expandable rows performance test</h1>
+
+      {isActive ? (
+        <SpaceBetween size="s">
+          <Button
+            onClick={() => {
+              actions.setExpandedItems(allExpandableInstances);
+              console.time('expand-all');
+              requestAnimationFrame(() => console.timeEnd('expand-all'));
+            }}
+          >
+            Expand all
```

Comment:
Minor: I don't see any value in having the expand and collapse all buttons for the table without expandable rows. Should we remove the buttons for that?